bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_run_card.dat

```diff
@@ -0,0 +1,161 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#                                                                      *
+#   Some of the values of variables can be list. These can either be   *
+#   comma or space separated.                                          *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+ 150 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+ 100 = nevt_job! Max number of events per job in event generation. 
```

Comment:
Please change this to -1 default settings

Reply:
Again I copied the values from the 2016 cards. Is this strictly required? and what changes are expected by switching to -1? We published the 2016 analysis a few months ago.

Reply:
It won't make any difference to the physics, it changes how MadGraph handles computations

Reply:
Ok, then I would prefer to go head with these settings, given that each gridpack takes ~50 hours to finish and we have 8 gridpacks in total. We do not really want to delay the things ~3 weeks more if physics does not change at all

Reply:
Sorry for coming back to this very late (it's okay to just ping me through Skype... sorry again)

@agrohsje is this fine for you? I am not really aware of "how" it will be different (like whether it will screw up the McM central cores for production or not, but guess it won't, no?).